style_editor.html

```diff
@@ -2496,17 +2499,19 @@
   x = Math.pow(x, 1.0/2.2);
   return Math.round(x * 255);
 }
+
 function mkcolorbutton(name, r, g, b) {
   r = mapcolor(r);
   g = mapcolor(g);
   b = mapcolor(b);
-  var color = "rgb("+r+","+g+","+b+")";
-  if (r == 0 && g == 0 && b == 0) color = "white";
-  return "<input type=button style='border: 1px solid black;padding:8px;color:black;background:"+color+"' class=btn onclick='SetTo(\""+name+"\")' value='"+enc(name)+"'>\n";
+  var text = (name === "Black" || name === "Blue") ? "white" : "black";
```

Comment:
I checked your copy, but I didn't see what this looks like.

Reply:
just checked it in now